lib/beaker/host/unix/pkg.rb

```diff
@@ -17,28 +17,31 @@ def check_for_command(name)
     end
   end
 
-  def check_for_package(name)
+  def check_for_package(name, opts = {})
+    opts = {:accept_all_exit_codes => true}.merge(opts)
+    opts[:prepend_cmds] ? pc = opts.select {|k,v| k == :prepend_cmds} :
```

Comment:
Agreed.  That is exactly what I am doing below in the (install | uninstall | update)_package methods,  and hence my question about normalizing the method calls in pkg.rb.  I am glad you are in favor of normalizing to use execute.  It does of course mean we keep the changes to the execute method to pass the :prepend_cmds option through.  I am assuming you are ok with that change?